lib/app.js

```diff
@@ -30,26 +30,28 @@ app.use(function* () {
     this.throw("Expected a source file", 400);
   }
   const sourceFilePath = parts.file.source.path;
+  const sourceDirPath = path.dirname(parts.file.source.path);
 
-  if (!parts.file.caniuse) {
-    this.throw("Expected caniuse file", 400);
-  }
-  const caniuseFilePath = parts.file.caniuse.path;
-
-  if (!parts.file.w3cbugs) {
-    this.throw("Expected w3cbugs file", 400);
-  }
-  const w3cbugsFilePath = parts.file.w3cbugs.path;
+  const quiet = parts.field.quiet !== undefined ? "--quiet" : "";
```

Comment:
Hmm I was thinking of using query string for these and leaving the multipart body for the actual uploads. Should be able to check `this.query.quiet` etc.